template.yaml

```diff
@@ -322,6 +322,27 @@ Resources:
             Method: post
             RestApiId: !Ref ScientificIndexApi
 
+  FetchNviPeriodsHandler:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: nvi-rest
+      Handler: no.sikt.nva.nvi.rest.fetch.FetchNviPeriodsHandler::handleRequest
+      Role: !GetAtt NvaNviRole.Arn
+      Timeout: 600
+      MemorySize: 1536
+      AutoPublishAlias: live
+      Environment:
+        Variables:
+          ALLOWED_ORIGIN: '*'
+          API_HOST: !Ref ApiDomain
+      Events:
+        ApiEvent:
+          Type: Api
+          Properties:
+            Path: /period
```

Comment:
Unsure about endpoint path name, but since fetch period endpoint will most likely be period/{periodIdentifier}, so it is ok to use GET /period for fetching all periods